org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/utils/FilesUtils.java

```diff
@@ -262,6 +264,21 @@ public static Path getPath(String uri) {
 		return Paths.get(uri);
 	}
 
+	/**
+	 * Returns true if the provided path exists.
+	 * Returns false otherwise.
+	 * 
+	 * @param path the path to check
+	 * @return true if the provided path exists and false otherwise
+	 */
+	public static boolean pathExists(String path) {

```

Comment:
I wonder if you could use the existing FilesUtils.getPath to avoid creating a new utility method?

Reply:
Fixed. I removed this method and now I am using `Files.notExists(FilesUtils.getPath(c));` and `Files.exists(FilesUtils.getPath(c));`